x-pack/plugins/apm/server/lib/search_strategies/queries/query_percentiles.ts

```diff
@@ -23,10 +23,9 @@ import { SIGNIFICANT_VALUE_DIGITS } from '../constants';
 export const getTransactionDurationPercentilesRequest = (
   params: SearchStrategyParams,
   percents?: number[],
-  fieldName?: FieldValuePair['fieldName'],
-  fieldValue?: FieldValuePair['fieldValue']
+  termFilters?: FieldValuePair[]
```

Comment:
Same here. Using destructuring will make it easier to read

Reply:
To do this consistently for all queries would require to refactor more files than the ones already touched in this PR, unrelated to the feature. I added an item to the meta issue to pick it up in a follow up https://github.com/elastic/kibana/issues/109220

Reply:
Doing in a follow-up sgtm. Thanks